### ISSUE TITLE
Frequent timeouts for CI / base (3.9)

### ISSUE THREAD

**Issue by dweindl** [ICB-DCM/pyPESTO#1077]
e.g. https://github.com/ICB-DCM/pyPESTO/actions/runs/5187107504/jobs/9349064642?pr=1073

Increase timeouts or speed up tests?

**Comment by dweindl** [comment]
Still happening. Now also for julia tests (e.g. https://github.com/ICB-DCM/pyPESTO/actions/runs/5959628067/job/16166590355)

**Comment by dweindl** [comment]
Also for the mac tests: e.g. https://github.com/ICB-DCM/pyPESTO/actions/runs/6034032186/job/16371678336

**Comment by PaulJonasJost** [comment]
base should be faster in newer version now.

**Comment by dweindl** [comment]
> base should be faster in newer version now.

Times out less frequently, but still does occasionally: https://github.com/ICB-DCM/pyPESTO/actions/runs/6697723646/job/18198209071

**Comment by dweindl** [comment]
While I think it's good to keep the test suite as compact as possible, I would also suggest to just extend the time limits. With re-running all the timed-out tests the current limits waste more time than they save.

**Comment by PaulJonasJost** [comment]
Yes, think that is a good idea. What I also noticed is that the tests seem to run out if too many actions run at the same time. Partially this might be due to two reasons:
* Merge develop into most/all open PRs
* Multiple commits into a single PR as this triggers an action but does not cancel the previous one

**Comment by dweindl** [comment]
Also for notebook tests, e.g. https://github.com/ICB-DCM/pyPESTO/actions/runs/6903646092/job/18782766099
